ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -6,7 +6,7 @@
 shutdown_interfaces_towards_undeployed_peers: true
 
 # Enable vlan aware bundles
-evpn_vlan_aware_bundles: true
+evpn_vlan_aware_bundles: false
```

Comment:
please don't change existing molecule tests. This will change the coverage of all the existing tests based on the same file. I suggest you create a new separate test set similar to EVPN_MULTICAST_TESTS (only configure the stuff you need and with the least amount of devices required). Two L2 EVPN gateways, one with vlan-aware-bundle and one without should be enough in this case.

Reply:
Will do - I have changed this back to it's original setting of `true`. Thanks!